diff --git a/configure.ac b/configure.ac
index c334f32041..7c73ce0a85 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,9 +4,7 @@
 # Configure script
 #
 
-m4_ifdef([m4_esyscmd_s],
-	[AC_INIT([NetXMS], m4_esyscmd_s([git describe --always --long | sed 's/^Release-//' | sed 's/-g.*//' | sed 's/-/./;/^[0-9][0-9]*\.[0-9][0-9]*$/ s/$/.0/']), [bugs@netxms.org])],
-	[AC_INIT([NetXMS], [3.9.0], [bugs@netxms.org])])
+AC_INIT([NetXMS], [3.9.361], [bugs@netxms.org])
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_HEADERS(config.h)
 AM_INIT_AUTOMAKE
